### DISCUSSION TITLE
资源占用问题

### DISCUSSION THREAD

**Discussion by violenttamper** [violentmonkey/violentmonkey#1274] [Q&A] [answered]
今天打开某网站时，变得有些卡，打开了资源管理器，CPU有些爆红了，排查了一下，发现关了violentmonkey扩展就好了，不知道怎么回事？使用的是Vivaldi浏览器 版本3.8.2259.37 (Stable channel) （64 位），violentmonkey扩展版本2.13.0

**Comment by violenttamper** [comment, marked as answer]
我排查了下，是某个脚本的问题，请问，脚本问题会影响资源占用吗？

**Reply by violenttamper** [reply]
对比了下tampermonkey扩展，它加载了那个问题脚本后也导致了占用过高。不过violentmonkey扩展开启了8-9个脚本时，在只开一个浏览器的一个视频页面时偶尔CPU会占用过多，请问这是什么原因？

**Reply by tophf** [reply]
No one will be able to help you based on this description. The only way to solve this problem is to investigate what happens on your computer in devtools.

**Reply by violenttamper** [reply]
我排查出来了，大概率是脚本的问题。
